src/Init/Data/Repr.lean

```diff
@@ -13,11 +13,24 @@ open Sum Subtype Nat
 
 open Std
 
+/--
+A typeclass that specifies the standard way of turning values of some type into `Format`.
+
+When rendered this `Format` should be as close as possible to something that can be parsed as the
```

Comment:
"As close as possible" is a bit tricky :-) Is it a bug if it doesn't round-trip?

Reply:
Leo mentioned that it is "the moral obligation" but it is not strictly considered a bug.

Reply:
OK, this formulation is good then :)